### DISCUSSION TITLE
v1.10.0 - Maintenance Release

### DISCUSSION THREAD

**Discussion by iwalton3** [jellyfin/jellyfin-desktop#622] [General]
This release upgrades Jellyfin Media Player to work better with the latest version of the Jellyfin server. Numerous features and improvements have also been added by the community.

Note for Flatpak users: Flatpak builds have been held up for at least 4 hours as of writing this. I have pushed the build and it'll get processed as soon as the job queue is unstuck.

  * Update web client to 10.9.1.
  * Switch to native Jellyfin Trickplay support.
  * Update mpv version to latest git version.
  * Misc fixes and cleanup. (#603)
  * Fix trailing slash when adding server causes connection faliure. (#601)
  * Fix compatibility with mpv 0.38.0. (#597)
  * Fix appstream metadata (#592)
  * Show what's playing in MacOS's "now playing" (#580)
  * Add Apple silicon build. (#571)
  * Add tooltips and settings for MPV config. (#561)
  * Fix mpv deprecations. (#549)
  * Add support for FreeBSD. (#547)
  * Fix icon for Wayland. (#515)
  * Improve customization for subtitles. (#508)
  * Ensure that volume is always a Number. (#493)
  * Fix window restoring on opening with Hyprland. (#481)
  * Fix window switching monitor when exiting fullscreen. (#478)
  * Fix skipintro plugin client freeze. (#456)
  * Set max audio channels for transcode profile based on JMP settings. (#445)
  * Add Default Playback Speed settings. (#435)
  * Remember playback speed. (#434)

If you have issues with media playback due to hardware decoding (green/purple or artifacts), you can [disable it using these instructions](https://github.com/jellyfin/jellyfin-media-player/wiki/Disabling-Hardware-Decoding).

About Windows Downloads:
 - The desktop installer requires administrator rights.
 - You can avoid warnings on files by unblocking the file in the properties dialog.
 - You can manually build a release [using these instructions](https://github.com/jellyfin/jellyfin-media-player#building-for-windows).

<hr /><em>This discussion was created from the release <a href='https://github.com/jellyfin/jellyfin-media-player/releases/tag/v1.10.0'>v1.10.0 - Maintenance Release</a>.</em>

**Comment by jtg567** [comment]
Upgraded to JellyfinMediaPlayer-1.10.0-windows-x64 and now my home screen is blank? I can only see content via the Search UI.

**Reply by GentleShadow13** [reply]
Confirmed this is happening with my self and a friend

**Reply by geirskogul** [reply]
Yeah, home screen is blank, this killed all functionality.  Complete uninstall/folder delete/reinstall, no dice.  Re-login, no dice.  We have about 25 users and this is unusable :(

**Reply by GentleShadow13** [reply]
@geirskogul uninstalling and reinstalling 1.9.1 worked for me for now.

**Reply by geirskogul** [reply]
Yeah, 1.9.1 fixed it back for me.  Strange.  All the media was searchable but the home screen was blank and the library tabs inaccessible on 1.10.  Thank you @GentleShadow13 for the rollback tip.    Winx64 I should specify.

**Reply by GentleShadow13** [reply]
Same on the x64 :)

**Reply by geirskogul** [reply]
Doesn't work on a fresh windows VM either, for testing.  Can connect but same, blank screen.  Just if anybody wants info.

**Reply by addmix** [reply]
Yup, same issue. Blank dashboard, no way to view any libraries or videos.
Windows 11 x64

**Reply by GreyingRaven** [reply]
I upgraded my server from 10.8.11 to 10.9.1, and it fixed the issue.
I have my Jellyfin server set up on a Linux server Ubuntu 22.04.2 LTS using a docker-compose file and player on Windows10 x64.

**Reply by Nevaah** [reply]
Same issue Win11 x64.
Will look into upgrading server and edit as I can read that could fix it :)
*EDIT* Yup, server upgrade to 10.9.1 fixes it too (Running Synology container)

**Reply by Tatoon1983** [reply]
How do you upgrade the server to 10.9.1 in Ubuntu? The latest version in the repos is 10.8...

**Reply by Heropool** [reply]
anyone know how to update server on asustor nas?

**Reply by KaedeusGrenfell** [reply]
I had the same issue. Upgraded my Server on my TrueNAS Scale machine. Restarted the app on my Windows PC and display has come good. 

TrueNAS Catalog
Name: jellyfin
App Version: 10.9.1
Chart Version: 1.2.10

**Reply by GentleShadow13** [reply]
> I upgraded my server from 10.8.11 to 10.9.1, and it fixed the issue.
> 
> I have my Jellyfin server set up on a Linux server Ubuntu 22.04.2 LTS using a docker-compose file and player on Windows10 x64.
> 
> 

This also worked on my windows machine.

**Comment by Lauvizh** [comment]
Hello (thanks for this amazing app),
Had this spash screen when opening `JellyfinMediaPlayer-Intel.dmg` on a MacBookPro Apple Intel (i7)
<img width="300" alt="image" src="https://github.com/jellyfin/jellyfin-media-player/assets/7997356/859cf8b7-8895-4c70-b919-a4a112e33c63">

The` JellyfinMediaPlayer-AppleSilicon.dmg` did the empty home screen on a M1 Mac mini

Server : Version 10.8.13
Run on Synology package from [@stevenliuit](https://github.com/stevenliuit)

**Reply by tigerwoo** [reply]
me too，soonma 14.4， intel core i5

**Reply by Lauvizh** [reply]
I switched back to 1.9.1 it fixed it, and i'am waiting on the synology package to be updated.

**Reply by elkenders** [reply]
Yep same here using the Intel Package with Rosetta on an M1 Ultra. The Native Silicon package completely crashes on launch with a "Namespace DYLD, Code 4 Symbol missing". Some dependency but I've got everything fully updated and tried scrubbing cache etc.

**Reply by mikewesten** [reply]
No linux focal release?

**Reply by lukaslindnermusic** [reply]
I have the same splash screen on an iMac 2020 with 3,8 GHz 8-Core Intel Core i7, running the latest jellyfin docker image (10.9.1) on a Synology. I already deleted the environment variable "DOTNET_SYSTEM_GLOBALIZATION_INVARIANT" as mentioned in the installation blog post, and in the browser it is working fine.
Clicking on "reload" doesnt do anything, as well as clicking on "logfile". "support page" just opens the jellyfin website.
downgraded to Jellyfin Media Player 1.9.0 now.

**Reply by lukaslindnermusic** [reply]
Similarly, on my Macbook Air M1, Jellyfin Media Player with the Apple Silicon build just crashes when I try to open it.
with the Intel Build opening under Rosetta, I get the same splash screen as with the Intel processor iMac.

**Comment by kradear** [comment]
![image](https://github.com/jellyfin/jellyfin-media-player/assets/122838197/c05deca9-2022-42ae-98af-9400025a7aba)

Since updating I'm having trouble with the subtitles. When I play something, they automatically load all crammed to the side. I have to turn them off and re-select them for them to display correctly. Anyone having such problems?

**Comment by majjuss** [comment]
any specific reason that there isn't a deb file for "focal" any more? Can I use another one of the provided deb files?
I'm on Linux Mint 20.3 Cinnamon...
Any help would be appreciated!
